README.md

```diff
@@ -90,17 +88,22 @@ client = IronMQ::Client.new(project_id: "57a7b7b35e8e331d45000001", keystone: ke
 ### Get Queues List
 
 ```ruby
-list_queues = ironmq.queues # => [#<IronMQ::Queue:...>, ...]
+queues = ironmq.get_queues # => [#<IronMQ::Queue:...>, ...]
+# alias:
+#   Client#queues
 ```
 
 --
 
-### Get a Queue Object
+### Make a Queue Object
 
 You can have as many queues as you want, each with their own unique set of messages.
 
 ```ruby
-queue = ironmq.queue('my_queue')
+# pass queue name
+queue = ironmq.make_queue('my_queue') # => #<IronMQ::Queue:...>
```

Comment:
I like the old syntax better, just ironmq.queue('my_queue'). This isn't necessarily making a queue so the method is misleading.